source/accounts/accounts_and_projects.rst

```diff
@@ -0,0 +1,251 @@
+   preferred if you have a smartphone.
+#. Request a Common Access Card (CAC)
+
+NOAA will conduct a security investigation that takes 6-12 weeks.
+This will include a fingerprint check, background check, and DOC
+public trust investigation.
+
+Foreign national participants have special hoops to jump through.
+
+Once your account request has been vetted by the security team, you
+will receive an email from the accounts team and will be able to
+proceed to request access to projects allocated to systems.
+
+.. note::
+
+    NOAA RDHPCS systems use a **long user name** which is usually the same as your
+    NOAA email user name. These pages frequently refer to this long user name as
+    *First.Last*.
+
+.. attention::
+
+   You **must** access your NOAA.gov email account regularly.  **All**
+   communications about the NOAA RDHPCS program will be sent to your
+   NOAA.gov email account.
+
+.. _project_request:
+
+Request access to RDHPCS projects
+=================================
+
+.. attention::
+   You must be able to access the `NOAA Accounts Portal`_ before
+   proceeding.
+
+Once you have an RDHPCS account, you can request access to projects
+that have allocations on systems.
+
+#. Consult with your supervisor and colleagues to identify the
+   project(s) to request access to.
+#. Access the `Account Information Management`_ to request
+   project access.  **Provide a clear reason for your request.**
+#. The principal investigator (PI) of the project must approve your
+   project request.  They will be automatically notified and regularly
+   reminded of open requests.
+#. Once your project request is approved, a welcome email will be sent
+   to you.
+
+Requesting access to MSU-HPC systems
+====================================
+
+The Hercules and Orion systems comprise MSU-HPC, managed by
+Mississippi State University. Follow these steps to get an Account for
+MSU-HPC.
+
+General Access Requirements
+---------------------------
+
+-  All users, regardless of citizenship, follow the same process to
+   receive MSU-HPC access.
+-  NOAA's RDHPCS users will need to use MSU’s HPC Account Management
+   System and Process. RDHPCS Portfolio Managers have access to MSU's
+   Account Management Tool.
+-  The PfM must have allocations to use the MSU-HPC system.
+-  MSU's Account Management system requires user authentication. PIs
+   and Portfolio Managers must maintain an active MSU account to
+   manage their projects online.
+
+ .. note::
+
+   The designated PI or Portfolio Manager (PfM) must request that a
+   MSU user account be created and the user assigned to their project.
+
+
+Complete the following steps for MSU-HPC access.
+
+- Collaborate with a NOAA research lab and be associated with an
+  active NOAA research project. Each project has an assigned Principal
+  Investigator (PI) who is responsible for the project and the project
+  members. The PI or PfM requests both project assignment and account
```

Comment:
Should we have the definition of the acronyms (PI, PfM) aligned to the first references?  Seems strange to have the acronyms used and then defined when referenced again later on...